apps/consent/app/api/csrf-invalid/route.ts

```diff
@@ -0,0 +1,11 @@
+import type { NextApiRequest, NextApiResponse } from "next";
```

Comment:
I know it was in the hydra consent code but I'm wondering if this is something we should be keeping. feels like test or unused code

Reply:
Yes, it's unused now; removed.